packages/cli/src/scripts/interfaces.ts

```diff
@@ -0,0 +1,39 @@
+import ZosPackageFile from '../models/files/ZosPackageFile';
+
+export interface InitParams {
```

Comment:
Could some of these be non optional?

Reply:
Yes, most of them. I was planning to refactor these interfaces at the end of the port :smile: